kodilla-collections-advanced/src/test/java/com/kodilla/collections/adv/immutable/BookTest.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.collections.adv.immutable;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class BookTest {
+    private final Book alchemik = new Book("Paulo Coelho", "Alchemik");
+
+    @Test
```

Comment:
BEZ SENSU. Po co testujesz, czy Java dziala? Nie testujemy konstruktorow, geterow, toString, equals, hashCode, setterow itp ...

Reply:
Usun "glupie testy" - ja wiem, ze tu nie dojdziesz do takiego pokrycia kodu, jaki jest wymagany. Zaakceptuje wtedy - ale tylko logike biznesowa testujemy, czyli serwisy, mappery, metody uzytkowe. Pamietaj, ze kazdy test to czas - chcemy miec kompromis -> czas budowania z testowaniem kontra pokrycie kodu testami. Bez sensu testowac cos, co na 100% dziala :)